### DISCUSSION TITLE
O3DE Release 22.10.0: Known Issues

### DISCUSSION THREAD

**Discussion by chanmosq** [o3de/o3de#12129] [General]
With the upcoming O3DE 22.10.0 release, we are collecting all of the known issues to list in the release notes. In this discussion, you can add known issues, including: 
- Bugs and limitations (like unsupported platforms, performance issues, etc.)
- Issues regarding new features for this release
- Issues regarding features from prior release, but newly found

Below, please leave a comment to add a known issue and include the following:
-  Short sentence/phrase that summarizes the issue. 
- Additional details to further explain the issue.
  - Which platform this issue pertains to.
- How to work around the issue, if applicable. (If this is long, I suggest hosting the workaround instructions in another place and linking to it. For example, you may write the instructions in the associated GitHub Issue.)
- A link to the GitHub Issue, if one exists. 

---
_Note: Additionally, ensure that new features for the release (or deprecated features) are listed in the [O3DE 22.10.0 Feature List](https://github.com/o3de/sig-release/blob/main/releases/22.10.0/22100%20feature%20list.md). You can add them in by creating a pull request against that page, or leave a comment in #12062._

**Comment by L4exp** [comment]
Few significant bugs:

#9448 

UDIM(UV tiles issue) - problem with displaying
----------------------------------------------------------
 #7519 

On AMD(MESA) all objects are dark everywhere(game mode, editor mode, terrain). Textures, which linked in object's material
actually not displayed(white color only - if gain intencity on sun e.g.). On Nvidia - from issue description, in game mode it behaves differently(in #7519) described.

**Comment by Kadino** [comment]
Crash logs are not available on Linux: https://github.com/o3de/o3de/issues/10346

**Comment by NeilGitGud** [comment]
[GHI - 508](https://github.com/o3de/o3de-atom-sampleviewer/issues/508)

Running AtomSampleViewer.exe "_fulltestsuite_.bv.luac" crashes when run, but individual samples all run correctly.

**Comment by amzn-changml** [comment]
VS2022 slower build times in some situations compared to VS2019: https://github.com/o3de/o3de/issues/12257

**Comment by hultonha** [comment]
Physics Known Issues

- Do not set the Cylinder Collider radius or height to zero as this will cause a crash. This is a known issue and will be fixed in development but currently exists in the stabilization branch (see for more details https://github.com/o3de/o3de/issues/12147).

Viewport Known Issues

- The Component Mode Switcher may suffer from instability on Linux. If this is experienced it is possible to simply disable the Switcher in the Settings Registry by adding...

```
{
    "Amazon": {
        "Preferences": {
            "Editor": {
                "ComponentSwitcherEnabled": false
            }
        }
    }
}
```

Character Known Issues

- Do not include `.` character in names for animations in you DCC tool as this can fail at import time (see https://github.com/o3de/o3de/issues/12342 for more details).
- Ensure a level is opened before accessing the scene settings tool (see https://github.com/o3de/o3de/issues/12414 for more details).

**Comment by amzn-phist** [comment]
[#11558](https://github.com/o3de/o3de/issues/11558)

Audio
- There's known log spam that occurs during shutdown of GameLauncher.  These warnings can be safely ignored, as it pertains to trying to clean up objects that were already deleted.

**Comment by forkercat** [comment]
Prefab Known Issues
- Required to save level twice after creating and deleting prefabs. 
- Some warnings like "Some of the patches were not successfully applied" might show up in the console after prefab operations.
    - For deletion, it has been fixed in development but that fix was before stabilization cutoff. [#11756](https://github.com/o3de/o3de/issues/11756)
    - For creation, it will be fixed in development. [#12013](https://github.com/o3de/o3de/issues/12013)

**Comment by chanmosq** [comment]
All of these issues have been added to the release notes. If you have more known issues, please leave a comment in https://github.com/o3de/o3de.org/pull/2021
